.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Issue with ASP.NET Core
```

Comment:
Should fix the contacts.

Reply:
> Should fix the contacts.

These are in addition to the bug, feature and question links pertaining scaffolding and msidentity scaffolding. Take a look at https://github.com/dotnet/efcore/issues/new/choose for reference. We can just remove them but no harm in giving devs the options to report those issues.